tools/update-rules.js

```diff
@@ -50,18 +50,38 @@ const rulesTableContent = categories.map(category => `
 |:---|:--------|:------------|
 ${
   rules
-    .filter(entry => entry[1].meta.docs.category === category)
+    .filter(entry => entry[1].meta.docs.category === category && !entry[1].meta.deprecated)
     .map(entry => {
       const name = entry[0]
       const meta = entry[1].meta
       const mark = `${meta.docs.recommended ? STAR : ''}${meta.fixable ? PEN : ''}${meta.deprecated ? WARN : ''}`
       const link = `[${name}](./docs/rules/${name}.md)`
-      const description = (meta.docs.description || '(no description)') + (meta.deprecated ? ' - (deprecated)' : '')
+      const description = meta.docs.description || '(no description)'
       return `| ${mark} | ${link} | ${description} |`
     })
     .join('\n')
 }
-`).join('\n')
+`).join('\n') + `
+### Deprecated
+
+> - :warning: We are going to remove deprecated rules in future. Please migrate to its replaced rules.
```

Comment:
`We're going to remove deprecated rules in the next major release. Please migrate to successor/new rules.`